src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -87,7 +87,7 @@ public class ReportGenerator {
     private static final String LOG_ERROR_CLONING_OR_BRANCHING = "Exception met while cloning or checking out.";
     private static final String LOG_UNEXPECTED_ERROR = "Unexpected error stack trace for %s:\n>%s";
     private static final List<String> assetsFilesWhiteList =
-            Collections.unmodifiableList(Arrays.asList(new String[] {"favicon.ico"}));
+            Collections.unmodifiableList(Arrays.asList(new String[] {"favicon.ico", "title.md"}));
```

Comment:
Can we update the CLI syntax reference in the User Guide to include the new `title.md` file? 
Should be this page: https://reposense.org/ug/cli.html